### ISSUE TITLE
Create support page on website and remove "send email to support"

### ISSUE THREAD

**Issue by lorenzoPrimi** [ooni/probe#1227]
"Speaking of settings, how is the "send email to support" option actionable/useful? Do we want to consider removing it?

In almost all cases, it seems that users send us empty emails without any actionable information. Furthermore, we don't really have the in-house resources to follow-up with every single empty email that we receive."
@agrabeli 

In my opinion a form (web or native) would be a better solution, let's discuss it.

**Comment by agrabeli** [comment]
@lorenzoPrimi I agree, I think having a form is a great idea.

The form could include required fields and unless the user fills out those fields, they cannot submit the form (to avoid receiving empty, non-actionable messages). 

Perhaps the form could start off with a drop-down menu, where the user first specifies why they're reaching out. This drop-down could include fields like "bug report", "feedback", "need help", etc. Then we could have an open-ended space where they could write and describe the issue. However, I guess some research is required before we start designing what this form would look like (and we probably also need to discuss the goals of this form).

**Comment by hellais** [comment]
We have decided that instead we are going to just create a page on ooni.org/support with instruction on how people can reach out to us for support.

**Comment by lorenzoPrimi** [comment]
@agrabeli edit "send email to support" from the copy to edit the button

**Comment by lorenzoPrimi** [comment]
@bassosimone pointed out that we need to start to actively using the beta channel for the app, and this task is a pre requisite for that, once we do more beta releases we need to have a better channel for users to communicate with us

**Comment by lorenzoPrimi** [comment]
@agrabeli ping me when this is done so I can remove that useless email feature from the app

**Comment by lorenzoPrimi** [comment]
This countly plugin noticed by @hellais https://count.ly/plugins/ratings-and-feedback
Can have the same problem as the current email system as it only ask for the satisfaction of the user, we will have the same issue with empty comment boxes.

This is more what we want https://count.ly/plugins/surveys
as we can ask targeted compulsory questions about which problem the user enconuntered
...but enterprise only!

**Comment by lorenzoPrimi** [comment]
We added a "Please describe the problem you are experiencing." string in the email and we decided this feature is not a priority nor needed now. 
Closing
